js/app.js

```diff
@@ -16,7 +16,8 @@ const ProjectCardsDetails = [
     id: 'work1',
     projectName: 'Hello-Microverse',
     projectInfo:
-      'Hello-Microverse-Project is mean to be an example of how to set up a simple project using git & github best practices.',
+        'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.Lorem Ipsum has been the industrys standard dummy text ever.',
+      //'Hello-Microverse-Project is mean to be an example of how to set up a simple project using git & github best practices.',
     ProjectIMG: [
```

Comment:
- [ ] It would be nice if you remove all the commented pieces of code from your app.js file to make the app.js file clean :+1: because you have added a lot of commented piece of code while making the requested changes :blush: